### PR TITLE
TasmoAdmin

### PR DESCRIPTION
**Pull request by tprelog**
[TasmoAdmin](https://github.com/reloxx13/TasmoAdmin/blob/master/README.md)

### CODE REVIEW

File: tasmoadmin.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "tasmoadmin",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/tprelog/iocage-tasmoadmin.git", 
+  "official": "false",
+  "properties": {
+    "allow_raw_sockets": "1",
+    "boot": "on",
```
Comment (miwi-fbsd) [inline tasmoadmin.json]: Please let this optional.

File: tasmoadmin.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "bpf": "on",
```
Comment (miwi-fbsd) [inline tasmoadmin.json]: Please remove it automated now.

File: tasmoadmin.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "dhcp": "1",
+    "vnet": "yes"
+  },
```
Comment (miwi-fbsd) [inline tasmoadmin.json]: Please remove it automated now.

File: tasmoadmin.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "name": "tasmoadmin",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/tprelog/iocage-tasmoadmin.git", 
+  "official": "false",
+  "properties": {
+    "allow_raw_sockets": "1",
+    "dhcp": "1"
+  },
+  "pkgs": [
+    "shells/bash",
```
Comment (miwi-fbsd) [inline tasmoadmin.json]: Hi,

categories can be removed iocage detects that automatically now, the pkg-list.json is also no needed. Could u please fix that for all ur submissions. Thanks.

Comment (tprelog) [reply tasmoadmin.json]: Fixed for all submissions!

Comment (miwi-fbsd) [reply tasmoadmin.json]: thanks!

**Comment by miwi-fbsd** [comment]
Please have a look at https://github.com/tprelog/iocage-tasmoadmin/issues/2

**Comment by tprelog** [comment]
pkg-list.json was leftover file. I have removed this from all my 11.3-RELEASE branches as requested

**Pull request merged**